content/kotlin/concepts/datetime/terms/toInstant/toInstant.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'toInstant()'
+Description: 'Converts a string representing an instant in ISO-8601 format, which includes both date and time components along with the time zone offset, into an Instant value.'
+Subjects:
+  - 'Mobile Development'
+Tags:
+  - 'Android'
+  - 'Kotlin'
+  - 'Date'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+**toInstant** converts a string representing an instant in ISO-8601 format, which includes both date and time components along with the time zone offset, into an Instant value.
+
+## Syntax
+
+```pseudo
+fun String.toInstant(): Instant
```

Comment:
Explain the parameters and fields in the syntax and write them down below the syntax.